packages/gatsby/src/commands/develop.ts

```diff
@@ -296,29 +319,16 @@ async function startServer(program) {
 
   /**
    * Set up the HTTP server and socket.io.
+   * If a SSL cert exists in program, use it with `createServer`.
    **/
-  let server = require(`http`).Server(app)
+  let server = program.ssl
+    ? https.createServer(program.ssl, app)
+    : new http.Server(app)
 
-  // If a SSL cert exists in program, use it with `createServer`.
-  if (program.ssl) {
-    server = require(`https`).createServer(program.ssl, app)
```

Comment:
If the `require` here was deliberately inlined you could do an `await import('https')` in the parent function and then access that in a closure.
But perhaps the inlining was just something that stuck around for no reason :)

Reply:
I don't think there is much reason to not import in a more typical fashion.